lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -379,6 +379,31 @@ class DecomposeAtenMatmulOp : public OpRewritePattern<AtenMatmulOp> {
 };
 } // namespace
 
+namespace {
+class DecomposeAtenTOp : public OpRewritePattern<AtenTOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenTOp op,
+                                PatternRewriter &rewriter) const override {
+    Value lhs = op.self();
+    int lhsRank = getTensorRank(lhs);
+    auto loc = op.getLoc();
+
+    // If both lhs and rhs ranks are 2 then map it to `aten.mm` op.
+    Value zero = rewriter.create<ConstantIntOp>(loc, rewriter.getI64IntegerAttr(0));
+    if (lhsRank < 2) {
+      rewriter.replaceOpWithNewOp<AtenTransposeIntOp>(op, op.getType(), lhs, zero,zero);
+    }
```

Comment:
Since this is identity case, you can simply replace the op with its operand. 
`replaceOp(op, lhs)`.

Reply:
Also, you can remove trivial braces here.